controllers/user.js

```diff
@@ -1,13 +1,14 @@
 const UserModel = require('../models/user')
 
 module.exports.get = async function get (req, res) {
-  const data = await UserModel.find({})
-  res.json({data})
+  const data = await UserModel.find({}).populate('office')
+  if (res.locals.role === 'admin') res.json({data})
+  else res.status(401).send('You do not have enough privileges for see this')
```

Comment:
in the case of 401 errors, is not necessary send a message, 401 always means unathorized

Reply:
I'm agree with the short if return without braces 

BUT, if the if sentence has an else i think is more readable use the the braces

if (res.locals.role === 'admin') {
  res.json({data})
} else {
  res.status(401).send('You do not have enough privileges for see this')
}

Reply:
Done, thanks for the feedbacks